package.json

```diff
@@ -23,6 +23,6 @@
   },
   "dependencies": {
     "brace": "^0.11.1",
-    "marked": "^0.6.2"
+    "marked": "^0.7.0"
```

Comment:
done separately in #170

Reply:
Yeah, there was some audit issues, I decided to fix while at it :)